guides/common/modules/ref_migrating-ansible-content.adoc

```diff
@@ -0,0 +1,30 @@
+[id="migrating-ansible-content_{context}"]
+= Migrating Ansible Content
+
+The upgrade from {RHEL} 7 to {RHEL} 8 includes an upgrade from Ansible Engine 2.9 to Ansible Core 2.12.
+
+If you have custom Ansible content such as playbooks, job templates inside REX, roles and collections on disk, and you relied on modules being delivered by the Ansible RPM on {Project}, you have to consider migrating your Ansible content.
+If you are using FQCN notation `namespace.collection.module` in your content, you can continue to use `ansible.builtin.*`, but everything else is missing in Ansible Core.
```

Comment:
This sounds like it depends on the notation, which I think is not the case, may need further rewrite